js/private/postinstall.bzl

```diff
@@ -0,0 +1,27 @@
+load("@aspect_bazel_lib//lib:run_binary.bzl", "run_binary")
+load("@aspect_bazel_lib//lib:copy_directory.bzl", "copy_directory")
+
+def postinstall(name, src, package_name):
```

Comment:
Is this macro worth the extra indirection, vs. in-line it, just print both targets in the generated build file?

Reply:
Good point. This started out as a rule rather than a macro, but given how simple it is it makes sense to inline it.